litmus/PTX/Proxy-Const-CoRR-cause-different-cta.litmus

```diff
@@ -1,5 +1,5 @@
-PTX fence
-"Two thread example with a constant proxy fence"
+PTX Proxy-Const-CoRR-cause-different-cta
```

Comment:
This is not a CoRR pattern, but a MP one (same applies for several tests below)

Reply:
Updated